js/layout-javascript/agenda-code.js

```diff
@@ -590,38 +660,57 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
     })
-    .on('click', '.agenda-date-selector li', function() {
-      // prevents clicking the active one
-      // prevents clicking the placeholder
-      if ($(this).hasClass('active') || $(this).hasClass('placeholder')) {
-        return;
-      }
+    .on('keydown', '.agenda-date-selector li', function(event) {
+      if (!$(this).hasClass('placeholder')) {
```

Comment:
1. Cache jQuery lookups https://github.com/airbnb/javascript/blob/master/README.md#jquery--cache Make sure this rule is not broken in the entire PR.
2. Use `||` to avoid multiple `if`s
3. To check two classes in one go, e.g. `active` and `placeholder`, you can use `.is('.active, .placeholder')`

Reply:
@tonytlwu, changed the logic